source/main.py

```diff
@@ -38,7 +38,7 @@ def parse_options(argv):
                         'and errors to syslog. --debug doesn\'t affect this option!')
     parser.add_argument("--limit", "-l", nargs=1, default=[None], metavar="[LIMIT]",
                         help='Limit task execution to specified host.')
-    parser.add_argument("--conf-dir", nargs=1, default=[None], metavar="conf_dir",
+    parser.add_argument("--conf-dir", "-C", nargs=1, default=[None], metavar="conf_dir",
```

Comment:
it's strange that it didn't require tens of changes in tests... maybe tets are failing and we're not aware?

Reply:
Thankfully (to you tbh since you were adding tests for --dry and --conf-dir) we use long version for those options.

Reply:
OK.. I thought that it's possible. We need an issue to check both (I'll create one)